File: public/sdk/java/ariana_enterprise_java/src/main/java/br/com/arianamoveis/enterprise/Webhooks.java
package br.com.arianamoveis.enterprise;
import java.util.*;
public class Webhooks {
  private final ArianaEnterpriseClient client;
  Webhooks(ArianaEnterpriseClient client){ this.client = client; }
  public Map<String,Object> test(String event, String message) throws Exception {
    return client.request("POST", "/webhooks/test", Json.of(Map.of("event",event,"message",message)));
  }
}

File: public/sdk/php/ariana-enterprise-php/src/Webhooks.php
<?php
namespace ArianaEnterprise;
class Webhooks { public function __construct(private Client $client) {} public function test(string $event = 'order_ack', array $data = []): array { return $this->client->request('POST','/webhooks/test',['event'=>$event,'data'=>$data]); } }
